transport/http/src/main/java/org/kaazing/gateway/transport/http/HttpAcceptor.java

```diff
@@ -593,6 +594,21 @@ public void removeBridgeFilters(IoFilterChain filterChain) {
         }
     }
 
+    private static HttpSubjectSecurityFilter newSubjectSecurityFilter(Logger logger) {
+        Class<HttpSubjectSecurityFilter> clazz = HttpSubjectSecurityFilter.class;
+        ServiceLoader<HttpSubjectSecurityFilter> loader = ServiceLoader.load(clazz);
```

Comment:
Other thing is that HttpSubjectSecurityFilter becomes external API. You could come up with smaller extension which may more maintainable from the API point of view. Either way, John needs to review this one.